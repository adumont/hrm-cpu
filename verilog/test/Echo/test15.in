18 93
22 fa
08 f3
18 6c
30 4e
38 f7
14 ca
48 c3
15 3a
40 34
24 8f
43 59
