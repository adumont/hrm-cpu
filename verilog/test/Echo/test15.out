93
fa
f3
6c
4e
f7
ca
c3
3a
34
8f
59
